frontend/src/utils/index.js

```diff
@@ -1,3 +1,6 @@
-export default function() {
-  console.log('arssss');
-}
+export const escapeRegExp = string =>
+  string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
```

Comment:
I'm wondering if there exists a library that does this type of security stuff for us out there.

Reply:
I did see Lodash has a method that could go here, going to test and report back.